cli-impl/src/main/java/org/aya/cli/literate/FaithfulPrettier.java

```diff
@@ -27,6 +29,30 @@ private static void checkHighlights(@NotNull ImmutableSeq<HighlightInfo> highlig
     });
   }
 
+  private static @NotNull ImmutableSeq<HighlightInfo> merge(
```

Comment:
Maybe move to `SyntaxHighlight`? It manipulates `HighlightInfo`, but `FaithfulPrettier` doesn't.

Reply:
See the updated faithful prettier, it now manages all highlights and problems of a single file, rather than the corresponding code part.